src/ConnectedMode/Resources.resx

```diff
@@ -289,4 +289,16 @@
   <data name="MuteWindowService_NotInConnectedMode" xml:space="preserve">
     <value>[Transition]Issue muting is only supported in connected mode</value>
   </data>
+  <data name="MuteIssuesService_Error_Caption" xml:space="preserve">
+    <value>Error</value>
+  </data>
+  <data name="MuteIssuesService_Error_CommentAdditionFailed" xml:space="preserve">
+    <value>Issue is resolved but an error occured while adding the comment, please refer to the logs for more information.</value>
+  </data>
+  <data name="MuteIssuesService_Error_FailedToTransition" xml:space="preserve">
+    <value>Unable to resolve the issue, please refer to the logs for more information.</value>
+  </data>
+  <data name="MuteIssuesService_Error_InsufficientPermissions" xml:space="preserve">
```

Comment:
I think there's specific name for the kind of permission level needed to be able to resolve issues on SQ/SC. We could add it to the error message so the user doesn't have to guess.